src/pages/api/register.ts

```diff
@@ -2,8 +2,27 @@ import { createAppRegisterHandler } from "@saleor/app-sdk/handlers/next";
 
 import { saleorApp } from "../../../saleor-app";
 
+const allowedUrlsPattern = process.env.ALLOWED_DOMAIN_PATTERN;
```

Comment:
Would consider moving `allowedUrlsPattern` decalaration/reading from env into `saler-app.ts`.
Not hard requirements, but rather my prefererence is to keep most of reading configuration from the ENV in the one file.

Reply:
Good, agree. Lets think about this after release. Technically it will be easy to do it in SDK.